client/.env

```diff
@@ -0,0 +1,2 @@
+EXPO_PUBLIC_SUPABASE_URL=https://rgvccomiqyiyjslyvmpe.supabase.co
+EXPO_PUBLIC_SUPABASE_PUBLIC_ANNO_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InJndmNjb21pcXlpeWpzbHl2bXBlIiwicm9sZSI6ImFub24iLCJpYXQiOjE2OTM1OTE4NjYsImV4cCI6MjAwOTE2Nzg2Nn0.MBLYmCYrzVgA2D7kgJCku0VodXAnWfpz5JihCRXv6lA
```

Comment:
I quickly checked and read that this is fine to be in github, but I dont understand how we keep the connection to the DB itself secure if all the credentials are publicly available? If you could explain this please

Reply:
The second key is a key that allowes you  to access the DB like an anonymous user. You dont access the DB directly, there is an api for  supabase called  Serverless Api which gives you an api that gives a feeling like you are directly acessing the DB, but you can configure the behaviour of it in the project's dashboard

Reply:
k cool, besides this all seems good to merge